docs/pbc/all/search/202410.0/base-shop/third-party-integrations/algolia/algolia.md

```diff
@@ -29,6 +29,7 @@ Your users can search for active concrete products by the following attributes:
 - Name
 - Description
 - Keywords
+- or any other attribute that your products have (when it's configured in Algolia Dashboard)
```

Comment:
Do the previous attributes not need to be configured in Algolia Dashboard?

Reply:
they're preconfigured by default by ACP Algolia App